solutions/USACO jan21-uddered-bronze.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: usaco-1083
+source: USACO Bronze 2021 January
+title: Uddered but not Herd
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob3_bronze_jan21.html)
+
+In this problem, we are asked to find the minimum number of times that Bessie must
```

Comment:
the official analysis explains it pretty well, so i don't think this explanation is necessary... anyone else have an opinion about this?